.gitignore

```diff
@@ -17,3 +17,8 @@ Thumbs.db
 
 # Text editor stuff
 *.swp
+
+# Dependencies installed by galaxy
+roles/wireguard
+roles/golang
+roles/elasticsearch
```

Comment:
Are they installed in `roles` for you?

For me, galaxy installs to `~/.ansible/roles`. Do you think you might find a way to accomplish the same for you? That way, we won't have to (and might potentially forget to) update gitignore every time deps change.

Reply:
Note that Elasticsearch can now actually be removed from deps!